src/nanoserver/1803/helix/amd64/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM mcr.microsoft.com/powershell:6.2.0-nanoserver-1803
+
+ENTRYPOINT C:\Windows\System32\cmd.exe
+
+RUN curl -SL --output python.zip https://www.nuget.org/api/v2/package/python/3.7.3 \
+    && md C:\Python C:\PythonTemp \
+    && tar -zxf python.zip -C C:\PythonTemp \
+	&& xcopy /s c:\PythonTemp\tools C:\Python \
+	&& rd /s /q c:\PythonTemp \
+    && del /q python.zip
+
+USER ContainerAdministrator 
+
+RUN setx /M PATH "%PATH%;C:\Python;C:\python\scripts" \
+ && setx /M PYTHONPATH "C:\Python\Lib;C:\Python\DLLs;" 
```

Comment:
_nit_ this should be indented to follow standard Dockerfile conventions and to be consistent within this Dockerfile.